frontend/src/components/core/Block/Block.tsx

```diff
@@ -57,8 +57,8 @@ const TextInput = React.lazy(() => import('components/widgets/TextInput/'))
 const TimeInput = React.lazy(() => import('components/widgets/TimeInput/'))
 
 type SimpleElement = ImmutableMap<string, any>
-type Element = SimpleElement | BlockElement
-interface BlockElement extends List<Element> { }
+type StElement = SimpleElement | BlockElement
```

Comment:
Element is the type for DOM elements. This makes it clearer that we're not talking about _those_ kinds of elements.